java-chassis-distribution/src/release/licenses/LICENSE-checker-framework

```diff
@@ -0,0 +1,413 @@
+The Checker Framework
```

Comment:
We can not ship the bundle of GPL licenses, we need to double check the Checker Framework license, if it provide the ASF license, we can just use it.

Reply:
I just using the maven license plugin to check the third party dependency，we just need to choose MIT License when specify it in the License file  ( I don't think we need to copy Checker Framework License here )

> (GNU General Public License, version 2 (GPL2), with the classpath exception) (The MIT License) Checker Qual (org.checkerframework:checker-qual:2.0.0 - http://checkerframework.org)